Jenkinsfile

```diff
@@ -429,7 +442,7 @@ pipeline {
 
                                             npm run lint
 
-                                            git -C src/brave checkout -q -
+                                            git -C src/brave checkout -q ${BRANCH_TO_BUILD}
```

Comment:
This change not needed - BUT I think it's good for clarity. Because it will ensure HEAD is not detached in case we needed to do debugging. Since this change is in `Jenkinsfile`, would it properly handle the situation where a SHA is pinned? or should we revert this single line change?

Reply:
you're right, have reverted for now